Jenkinsfile

```diff
@@ -275,6 +275,198 @@ stage("Unit Test") {
   }
 }
 
+stage("Build Tutorials") {
```

Comment:
looking at this file, I think you'd start benefiting from refactoring the pipelines into functions.

Reply:
any working example? I tried global functions but hit something similar to https://github.com/jenkins-docs/creating-a-pipeline-in-blue-ocean/issues/19